src/pretix/base/signals.py

```diff
@@ -202,6 +219,37 @@ def connect(self, receiver, sender=None, weak=True, dispatch_uid=None):
         super().connect(receiver, sender=None, weak=True, dispatch_uid=None)
 
 
+class Registry:
```

Comment:
This can certainly use some code documentation with a simple example in a docstring or soemthing like that to quicker understand the terminology used here around keys, accessors, items, …